src/main/java/com/faforever/client/chat/AbstractChatTabController.java

```diff
@@ -281,6 +285,7 @@ public void setReceiver(String receiver) {
 
   public void initialize() {
     mentionPattern = Pattern.compile("\\b(" + Pattern.quote(userService.getUsername()) + ")\\b", CASE_INSENSITIVE);
+    channelPattern = Pattern.compile("#([a-z][^#]+)\\b", CASE_INSENSITIVE);
```

Comment:
Constant as well. `mentionPattern` isn't because it needs to built with runtime information which is not the case for `channelPattern`

Also, I'd expect this pattern to be equal to what's being used in `channelUrlPattern`: `(#[a-z].+)` (or the other way around)